tests/test_integration_queue.py

```diff
@@ -0,0 +1,200 @@
+import json
+import os
+import requests
+
+from tests.utils import json_default, create_unlock_code_request, is_valid_uuid, create_unlock_code_activation, \
+    generate_uuid, create_unlock_code_revocation, create_tax_number_validity, create_send_est
+
+ERICA_TESTING_URL = os.environ.get("ERICA_TESTING_URL", "http://localhost:8000")
+
+
+class TestV2Ping:
+
+    def test_if_get_from_ping_then_return_pong(self):
+        response = requests.get(ERICA_TESTING_URL + "/v2/ping")
+        assert response.text == '"pong"'
+
+
+class TestV2UnlockCodeRequest:
+    endpoint = "/v2/fsc/request"
+
+    def test_if_post_with_full_data_then_return_201(self):
+        response = requests.post(ERICA_TESTING_URL + self.endpoint,
+                                 data=json.dumps(create_unlock_code_request(), default=json_default))
+        assert response.status_code == 201
+        response_content = response.json().split("/")
+        assert response_content[0] + "/" + response_content[1] == "fsc/request"
+        assert is_valid_uuid(response_content[2])
+
+    def test_if_post_without_full_data_then_return_422(self):
+        request_payload = create_unlock_code_request()
+        request_payload.clientIdentifier = None
+        response = requests.post(ERICA_TESTING_URL + self.endpoint,
+                                 data=json.dumps(request_payload, default=json_default))
+        assert response.status_code == 422
+
+    def test_if_get_existing_request(self):
```

Comment:
```suggestion
    def test_if_get_existing_request_then_return_200_and_response_with_correct_params(self):
```

Reply:
Maybe we can set those for all the test. WDYT?

Reply:
Done! Also renamed all other integration tests so that the functionality of the test can be directly read from the test name.